terraform/groups/ecs-service/profiles/live-eu-west-2/live/vars

```diff
@@ -19,4 +19,5 @@ piwik_url = "https://matomo.companieshouse.gov.uk"
 
 log_level = "info"
 node_env = "live"
-tz = "Europe/London"
\ No newline at end of file
+tz = "Europe/London"
+submit_validation_url = ""
```

Comment:
This value should not me missing as we will also need to call the xbrl validator in live.